app/controllers/events_controller.rb

```diff
@@ -14,6 +15,7 @@ def upcoming
 
   def create
     @event = Event.new(event_params)
+    authorize @event
```

Comment:
Might be nice to do a:

```ruby
before_filter :authorize, only: [:new, :create]
```

As I imagine we'll be authorizing more things in the future.